apps/src/code-studio/hiddenStageRedux.js

```diff
@@ -12,7 +12,7 @@ const SET_HIDDEN_STAGES = 'hiddenStage/SET_HIDDEN_STAGES';
 const UPDATE_HIDDEN_STAGE = 'hiddenStage/UPDATE_HIDDEN_STAGE';
 const UPDATE_HIDDEN_SCRIPT = 'hiddenStage/UPDATE_HIDDEN_SCRIPT';
 
-export const STUDENT_SECTION_ID = 'STUDENT';
+export const STUDENT_SECTION_ID = -2;
```

Comment:
might be worth a comment to explain this magic number - what is the significance of -2?

Reply:
Oh, good call, I will comment this up on the next PR (since I'd like to get this is in to unblock progress).  We needed a sentinel value less than 0 and -1 is already being used in some places to mean PENDING_NEW_SECTION_ID.